dcos_launch/config.py

```diff
@@ -542,3 +561,32 @@ def deduce_image_project(doc: dict):
         'type': 'boolean',
         'required': False,
         'default': False}}
+
+
+"""
+    Should these schemas exist at all? Input validation should be done at the terraform-dcos level to not duplicate the input validation
+    effort. However, some validation, like checking all input parameters are known, cannot be done in terraform-dcos (HCL)
+    Is it worth putting all these variables here just for that type of validation?
+"""
+ONPREM_TERRAFORM_DEPLOY_COMMON_SCHEMA = {
```

Comment:
all these "ONPREM" namespace designators are just adding clutter IMO, I think `AWS_TERRAFORM` is preferable to `AWS_ONPREM_TERRAFORM`

Reply:
@mellenburg Agreed. Any thoughts on my comment/question above that line?